.github/workflows/release.yml

```
@@ -38,17 +38,23 @@ jobs:
         with:
           tag-template: "{VERSION}"
           version-command: |
-            cat current-version.txt
+            cat galaxy.yml|grep version:|cut -d' ' -f2
```

Missing spaces for better yaml read:
```suggestion
            cat galaxy.yml | grep version: | cut -d' ' -f2
```